assets/docs/configuration/targets/http-full-example.hcl

```diff
@@ -43,15 +52,40 @@ target {
     dynamic_headers            = true
 
     # Optional. One of client credentials required when authorizing using OAuth2.
-    oauth2_client_id = env.CLIENT_ID
+    oauth2_client_id           = env.CLIENT_ID
 
     # Optional. One of client credentials required when authorizing using OAuth2.
-    oauth2_client_secret = env.CLIENT_SECRET
+    oauth2_client_secret       = env.CLIENT_SECRET
 
     # Optional. Required when using OAuth2. Long-lived token used to generate new short-lived access token when previous one experies.
-    oauth2_refresh_token = env.REFRESH_TOKEN
+    oauth2_refresh_token       = env.REFRESH_TOKEN
 
     # Optional. Required when using OAuth2. URL to authorization server providing access token. E.g. for Goggle API "https://oauth2.googleapis.com/token"  
-    oauth2_token_url = "https://my.auth.server/token"
+    oauth2_token_url           = "https://my.auth.server/token"
+
+    # Optional path to the file containing template which is used to build HTTP request based on a batch of input data
+    template_file              = "myTemplate.file"
```

Comment:
Come to think of it, I think we need two example template files in this directory, for the docs:

1. The template to preserve previous behaviour
2. Another example template that uses `range` - eg. the one we already use in integration tests

Reply:
Additionally, it would be nice to have the example contain `prettyPrint` and `env`